server/src/main/scala/com/cloudera/livy/LivyConf.scala

```diff
@@ -58,6 +58,8 @@ object LivyConf {
 
   val SERVER_HOST = Entry("livy.server.host", "0.0.0.0")
   val SERVER_PORT = Entry("livy.server.port", 8998)
+  val HTTP_REQUEST_HEADER_SIZE = Entry("livy.http.request.header.size", 131072)
+  val HTTP_RESPONSE_HEADER_SIZE = Entry("livy.http.response.header.size", 131072)
```

Comment:
Can we rename the configurations to "livy.server.request_header.size" and "livy.server.response_header.size", also it would be better to add some comments for these two configurations to describe why we have to change the default size.